From 2c70eb764a10c41dfb0026a7ac6d9223dbeeb5a1 Mon Sep 17 00:00:00 2001
From: Al Muthanna Athamina <almuthanna@chromium.org>
Date: Thu, 19 Aug 2021 10:00:26 +0200
Subject: [PATCH] [infra] Change loong-v comment to Loongson

Bug: v8:12100
Change-Id: I24a562d6e448b5f422ad3c891b00e15b9283a4ae
Reviewed-on: https://chromium-review.googlesource.com/c/v8/v8/+/3106385
Auto-Submit: Almothana Athamneh <almuthanna@chromium.org>
Reviewed-by: Michael Achenbach <machenbach@chromium.org>
Commit-Queue: Michael Achenbach <machenbach@chromium.org>
Cr-Commit-Position: refs/heads/main@{#76378}
---
 infra/mb/mb_config.pyl     | 2 +-
 infra/testing/builders.pyl | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/infra/mb/mb_config.pyl b/infra/mb/mb_config.pyl
index 4354d3fee06..7e587789603 100644
--- a/infra/mb/mb_config.pyl
+++ b/infra/mb/mb_config.pyl
@@ -194,7 +194,7 @@
       'V8 Linux - s390x - sim': 'release_simulate_s390x',
       # RISC-V
       'V8 Linux - riscv64 - sim - builder': 'release_simulate_riscv64',
-      # LOONG-V
+      # Loongson
       'V8 Linux - loong64 - sim - builder': 'release_simulate_loong64',
     },
     'tryserver.v8': {
diff --git a/infra/testing/builders.pyl b/infra/testing/builders.pyl
index 15accfbd727..f6079a86fc4 100644
--- a/infra/testing/builders.pyl
+++ b/infra/testing/builders.pyl
@@ -613,7 +613,7 @@
     ],
   },
   ##############################################################################
-  # Linux64 with LOONG-V simulators
+  # Linux64 with Loongson simulators
   'v8_linux64_loong64_rel_ng_triggered': {
     'swarming_dimensions': {
       'os': 'Ubuntu-18.04',
-- 
2.35.1

